code/datums/actions/items/stealth_box.dm

```diff
@@ -12,21 +12,21 @@
 	COOLDOWN_DECLARE(box_cooldown)
 
 ///Handles opening and closing the box.
-/datum/action/item_action/agent_box/Trigger(trigger_flags)
+/datum/action/item_action/agent_box/do_effect(trigger_flags)
 	. = ..()
 	if(!.)
 		return FALSE
```

Comment:
Should this still have a parent call, I noticed a lot had their's removed

Reply:
implants want their parent calls for comsigs yeah